Dockerfile

```diff
@@ -4,14 +4,15 @@ WORKDIR /srv/cartography
 
 ENV PATH=/venv/bin:$PATH
 RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3-dev python3-pip python3-setuptools openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3.8-dev python3-pip python3-setuptools openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev curl && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
-
 COPY . /srv/cartography
 
-RUN pip3 install -e . && \
-    pip3 install -r test-requirements.txt
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3.8 get-pip.py
```

Comment:
Can you add a quick comment on why this layer is needed so we remember why later on?

Reply:
Done